src/test/e2e.spec.ts

```diff
@@ -740,7 +803,7 @@ describe("e2e", () => {
       });
 
       const resp = await client.emitWithAck("rpc", deviceId, payload);
-      expect(resp).toEqual({ result: "ok" });
+      expect(resp).toEqual({ response: { result: "ok" } });
```

Comment:
what changed here?

Reply:
it's an older change but the test was not adjusted.